src/MCRun.jl

```diff
@@ -102,9 +122,9 @@ Returns probability to exchange configurations of two trajectories with energies
 at inverse temperatures `beta_1` and `beta_2`. 
 """
 function exc_acceptance(beta_1, beta_2, en_1, en_2)
-    delta_en = en_1 - en_2
+    d_en_acc = en_1 - en_2
     delta_beta = beta_1 - beta_2
-    exc_acc = min(1.0,exp(delta_beta * delta_en))
+    exc_acc = min(1.0,exp(delta_beta * d_en_acc))
     return exc_acc
 end
```

Comment:
more redefinition of variable names

Reply:
Could you please re-re-name here as well?